src/bin/pg_autoctl/cli_do_monitor.c

```diff
@@ -51,13 +55,13 @@ static CommandLine monitor_get_primary_command =
 				 keeper_cli_getopt_pgdata,
 				 keeper_cli_monitor_get_primary_node);
 
-static CommandLine monitor_get_other_node_command =
-	make_command("other",
-				 "Get the other node from the pg_auto_failover group of nodename/port",
+static CommandLine monitor_get_other_nodes_command =
+	make_command("others",
+				 "Get the other nodes from the pg_auto_failover group of nodename/port",
 				 " [ --pgdata ]",
 				 KEEPER_CLI_PGDATA_OPTION,
-				 keeper_cli_getopt_pgdata,
-				 keeper_cli_monitor_get_other_node);
+				 cli_do_monitor_get_other_nodes_getopts,
```

Comment:
only difference between two functions is addition --json parameter. Would it be possible add this in the existing getopts function instead of introducing regular one

Reply:
What about command that would then accept a `--json` option but fail to deliver JSON output? I guess we should have JSON output support everywhere now, rather than half baked easy-enough-to-parse-maybe output like we did at first. That sounds like another PR though.  Will work on that other PR separately.

Reply:
See https://github.com/citusdata/pg_auto_failover/pull/140.